visualization/dashboard/src/MLIDashboard/Controls/DatasetExplorerTab/DatasetExplorerTab.tsx

```diff
@@ -505,33 +518,42 @@ export class DatasetExplorerTab extends React.PureComponent<IDatasetExplorerTabP
             }
         }
         plotlyProps.data[0].customdata = this.buildCustomData(jointData, chartProps, cohort);
-        plotlyProps.data[0].hovertemplate = this.buildHoverTemplate(chartProps);
+        plotlyProps.data[0].hovertemplate = this.buildHoverTemplate(jointData, chartProps);
         return plotlyProps;
     }
 
-    private static buildHoverTemplate(chartProps: IGenericChartProps): string {
+    private static buildHoverTemplate(jointData: JointDataset, chartProps: IGenericChartProps): string {
         let hovertemplate = "";
+        const xName = jointData.metaDict[chartProps.xAxis.property].abbridgedLabel;
+        const yName = jointData.metaDict[chartProps.yAxis.property].abbridgedLabel;
         switch(chartProps.chartType) {
             case ChartTypes.Scatter: {
                 if (chartProps.xAxis) {
                     if (chartProps.xAxis.options.dither) {
-                        hovertemplate += "x: %{customdata.X}<br>";
+                        hovertemplate += xName + ": %{customdata.X}<br>";
                     } else {
-                        hovertemplate += "x: %{x}<br>";
+                        hovertemplate += xName + ": %{x}<br>";
                     }
                 }
                 if (chartProps.yAxis) {
                     if (chartProps.yAxis.options.dither) {
```

Comment:
the y axis and x axis case seem similar, consider refactoring the code out to a parameterized method, similarly the 
```
const xName = jointData.metaDict[chartProps.xAxis.property].abbridgedLabel;
const yName = jointData.metaDict[chartProps.yAxis.property].abbridgedLabel;
```